src/types/models.ts

```diff
@@ -249,12 +249,25 @@ export interface TimeBoundAccessCode extends ManagedAccessCodeBase {
   ends_at: string
 }
```

Comment:
Just curious, how come `created_at` is being added in this type, and not the base type?

Reply:
can add on base type